### ISSUE TITLE
knife-google master should be deployed to RubyGems

### ISSUE THREAD

**Issue by adamedx** [chef/knife-google#57]
@gmiranda23, can you update RubyGems with the latest version of this gem? Sync with @siddheshwar-more who is getting the specs green now.

**Comment by gmiranda23** [comment]
@adamedx can you grant me access to push that gem?  same handle as github.

**Comment by mansona** [comment]
I would like to +1 this because we started getting a pretty horriffic error when trying to use the 1.3.1 version: 

```
/opt/chefdk/embedded/lib/ruby/2.1.0/time.rb:325:in `_parse': no implicit conversion of nil into String (TypeError)
        from /opt/chefdk/embedded/lib/ruby/2.1.0/time.rb:325:in `parse'
        from /var/lib/jenkins/.chefdk/gem/ruby/2.1.0/gems/knife-google-1.3.1/lib/google/compute/zone_operation.rb:45:in `from_hash'
        from /var/lib/jenkins/.chefdk/gem/ruby/2.1.0/gems/knife-google-1.3.1/lib/google/compute/resource.rb:28:in `initialize'
        from /var/lib/jenkins/.chefdk/gem/ruby/2.1.0/gems/knife-google-1.3.1/lib/google/compute/creatable_resource_collection.rb:35:in `new'
        from /var/lib/jenkins/.chefdk/gem/ruby/2.1.0/gems/knife-google-1.3.1/lib/google/compute/creatable_resource_collection.rb:35:in `create'
        from /var/lib/jenkins/.chefdk/gem/ruby/2.1.0/gems/knife-google-1.3.1/lib/google/compute/creatable_resource_collection.rb:45:in `insert'
        from /var/lib/jenkins/.chefdk/gem/ruby/2.1.0/gems/knife-google-1.3.1/lib/chef/knife/google_server_create.rb:427:in `run'
        from /opt/chefdk/embedded/apps/chef/lib/chef/knife.rb:417:in `block in run_with_pretty_exceptions'
        from /opt/chefdk/embedded/apps/chef/lib/chef/local_mode.rb:38:in `with_server_connectivity'
        from /opt/chefdk/embedded/apps/chef/lib/chef/knife.rb:416:in `run_with_pretty_exceptions'
        from /opt/chefdk/embedded/apps/chef/lib/chef/knife.rb:213:in `run'
        from /opt/chefdk/embedded/apps/chef/lib/chef/application/knife.rb:139:in `run'
        from /opt/chefdk/embedded/apps/chef/bin/knife:25:in `<top (required)>'
        from /usr/bin/knife:40:in `load'
        from /usr/bin/knife:40:in `<main>'
```

Note the line number on zone_operation is wrong because i added some logging. Data was missing a startTime so it was effectively this line that was dying: https://github.com/chef/knife-google/blob/master/lib/google/compute/zone_operation.rb#L42

**Comment by paulrossman** [comment]
In the meantime and if you use bundler maybe pull from github?

**Comment by mansona** [comment]
I've just come across this same error again :sob: completely forgot that I posted here in March!! 

I was going to respond this back then but I got off the case, I'm not a rubyist so I don't know what `use bundler maybe pull from github` means. Can you elaborate? 

Also has there been any progress on this issue?

**Comment by adamleff** [comment]
Closing this out. `knife-google` is on Rubygems, and the recent work has been released as v2.1.0.
